### PR TITLE
Add Embedly and Slack-ImgProxy

### PR DESCRIPTION
**Pull request by caiosba**

### CODE REVIEW

File: crawler-user-agents.json [hunk 1]
```diff
@@ -864,5 +864,15 @@
...
+  },
+  {
+    "pattern": "Slack",
+    "addition_date": "2016/04/25",
+    "url": "http://slack.com"
+  },
+  {
+    "pattern": "Embedly",
+    "addition_date": "2016/04/25",
+    "url": "http://embed.ly"
```
Comment (monperrus) [inline crawler-user-agents.json]: could you change to http://support.embed.ly/ (the one in the header)?

File: crawler-user-agents.json [hunk 2]
```diff
@@ -864,5 +864,15 @@
     "pattern": "deadlinkchecker",
     "addition_date": "2016/02/08",
     "url": "http://www.deadlinkchecker.com"
+  },
+  {
+    "pattern": "Slack",
```
Comment (monperrus) [inline crawler-user-agents.json]: not specific enough, catches Slackware UAs, could you make it more specific

**Comment by caiosba** [comment]
PR updated as per your comments.

**Comment by monperrus** [comment]
thanks

**Pull request merged**